src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -133,19 +133,21 @@ theorem msb_eq_getLsb_last (x : BitVec w) :
   · simp [BitVec.eq_nil x]
   · simp
 
-@[bv_toNat] theorem getLsb_last (x : BitVec (w + 1)) :
-    x.getLsb w = decide (2 ^ w ≤ x.toNat) := by
-  simp only [Nat.zero_lt_succ, decide_True, getLsb, Nat.testBit, Nat.succ_sub_succ_eq_sub,
+@[bv_toNat] theorem getLsb_last (x : BitVec w) :
+    x.getLsb (w-1) = decide (2 ^ (w-1) ≤ x.toNat) := by
```

Comment:
Hmm, while this is mathematically more general, it is syntactically not. Can we keep the original statement, maybe named `getLsb_succ_last`, as a corollary of the new statement?

Reply:
I've added the other lemma tagged with `bv_toNat` as requested. Luckily, this is defeq, as `(w - 1) + 1 =def= w`, it's cheap to add.